cli/src/pcluster/validators/cluster_validators.py

```diff
@@ -892,7 +892,6 @@ def _validate(self, mount_dir: str):
             "/boot",
             "/dev",
             "/etc",
-            "/home",
```

Comment:
Resolving this because according to https://github.com/aws/aws-parallelcluster-cookbook/pull/2486/files#r1350507989 we are going to support /home in shared EBS volume

Reply:
yea at first I thought it wouldn't be that useful to mount as EBS, but better to just provide the flexibility